### PR TITLE
Minor fixes to logging scope examples

### PR DESCRIPTION
**Pull request by cijothomas**
1. Trying to show best practices, instead of plain strings. 
Will work with .NET ILogger owners to see if they can fix https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#log-scopes as well.

2. Also a minor fix to the ConsoleExporter to show "aka Body" for {OriginalFormat}

### CODE REVIEW

File: docs/logs/customizing-the-sdk/Program.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 using OpenTelemetry.Logs;
 using OpenTelemetry.Resources;
+using System.Collections.Generic;
```
Comment (reyang) [inline docs/logs/customizing-the-sdk/Program.cs:21]: ordering

File: docs/logs/customizing-the-sdk/Program.cs [hunk 2]
```diff
@@ -37,15 +38,18 @@ public static void Main()
...
+        logger.LogWarning("Hello from {name} {price}.", "tomato", 2.99);
+        logger.LogError("Hello from {name} {price}.", "tomato", 2.99);
 
         // log with scopes
-        using (logger.BeginScope("operation"))
-        using (logger.BeginScope("hardware"))
+        using (logger.BeginScope(new List<KeyValuePair<string, object>>
         {
-            logger.LogError("{name} is broken.", "refrigerator");
+            new KeyValuePair<string, object>("key1", "value1"),
```
Comment (reyang) [inline docs/logs/customizing-the-sdk/Program.cs:48]: Consider putting something that's more realistic (e.g. "store" = "Puyallup").

File: docs/logs/customizing-the-sdk/Program.cs [hunk 3]
```diff
@@ -37,15 +38,18 @@ public static void Main()
...
+            new KeyValuePair<string, object>("key", "value2"),
```
Comment (reyang) [inline docs/logs/customizing-the-sdk/Program.cs:49]:
```suggestion
            new KeyValuePair<string, object>("key2", "value2"),
```

File: src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs [hunk 4]
```diff
@@ -97,7 +97,7 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                                 ? new KeyValuePair<string, object>("OriginalFormat (a.k.a Body)", listKvp[i].Value)
                                 : listKvp[i];
 
-                            if (ConsoleTagTransformer.Instance.TryTransformTag(listKvp[i], out var result))
+                            if (ConsoleTagTransformer.Instance.TryTransformTag(valueToTransform, out var result))
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs:100]: This looks unrelated to the sample. I was going to update the PR title but I thought I would double-check you intended to include it first.

File: docs/logs/customizing-the-sdk/Program.cs [hunk 5]
```diff
@@ -37,15 +37,17 @@ public static void Main()
...
+        logger.LogInformation("Hello from {name} {price}.", "tomato", 2.99);
+        logger.LogWarning("Hello from {name} {price}.", "tomato", 2.99);
+        logger.LogError("Hello from {name} {price}.", "tomato", 2.99);
 
         // log with scopes
-        using (logger.BeginScope("operation"))
-        using (logger.BeginScope("hardware"))
+        using (logger.BeginScope(new List<KeyValuePair<string, object>>
+        {
+            new KeyValuePair<string, object>("store", "seattle"),
```
Comment (reyang) [inline docs/logs/customizing-the-sdk/Program.cs:47]:
```suggestion
            new KeyValuePair<string, object>("store", "Seattle"),
```

**Review by reyang — APPROVED** [review]
